web/src/LogStore.ts

```diff
@@ -315,13 +315,13 @@ class LogStore {
     // is uncertain. We should be more intelligent about sucking in events.
     for (let key in this.spans) {
       let candidate = this.spans[key]
-      if (candidate.manifestName != startSpan.manifestName) {
+      if (candidate.manifestName !== startSpan.manifestName) {
         continue
       }
 
       if (
         candidate.firstLineIndex > startSpan.firstLineIndex &&
-        (nextBuildSpan == null ||
+        (nextBuildSpan === null ||
```

Comment:
can we just make this !nextBuildSpan?

nextBuildSpan === null is almost never what you want, because that excludes undefined

Reply:
Yeah I wish this lint excluded cases where one of the operands is `null`.